Gemfile

```diff
@@ -2,7 +2,7 @@ require './prox_gem'
 
 prox_gem 'clock'
 prox_gem 'attribute'
-prox_gem 'null_attr'
+prox_gem 'subst_attr'
```

Comment:
Telemetry logger requires clock, attribute, subts_attr, and dependency, which is why I put them in. We're not currently using logger in the fixture library, but it seemed like infrastructure that should be available.